catalog/resource_online_table.go

```diff
@@ -64,6 +65,16 @@ func ResourceOnlineTable() common.Resource {
 			runTypes := []string{"spec.0.run_triggered", "spec.0.run_continuously"}
 			common.CustomizeSchemaPath(m, "spec", "run_triggered").SetAtLeastOneOf(runTypes).SetSuppressDiff()
 			common.CustomizeSchemaPath(m, "spec", "run_continuously").SetAtLeastOneOf(runTypes).SetSuppressDiff()
+
+			m["no_wait"] = &schema.Schema{
```

Comment:
Curious -- how useful would this be? My understanding is that it would fail most of the times if waiting for online table creation takes a few minutes. But if this helps some use cases, then I think it might be helpful to have this.

Reply:
I need to retest - we have a recent change when the product team added a native wait functionality, but it's not clear how much time we'll still spend on waiting